zerver/tests/test_signup.py

```diff
@@ -274,7 +274,7 @@ def test_register(self):
         with queries_captured() as queries:
             self.register(self.nonreg_email('test'), "test")
         # Ensure the number of queries we make is not O(streams)
-        self.assert_length(queries, 66)
+        self.assert_length(queries, 106)
```

Comment:
Yeah, I think we should do a backfill migration here and everywhere we can for the log_event calls.